iFixit/react-commerce
# Sentry: Add `fetch()` override

This adds an override for `fetch()` that adds [Sentry context](https://docs.sentry.io/platforms/javascript/guides/nextjs/enriching-events/context/) and Vercel error logging (via `console.error`).
All usages of `fetch()` are replaced with this override.

## QA

Make sure we get a context section when fetch requests fail.

CC @sterlinghirsh 

Closes #455
